### DISCUSSION TITLE
Security Onion users need help.

### DISCUSSION THREAD

**Discussion by kkdbudglf** [Security-Onion-Solutions/securityonion#10087] [Unsupported Versions]
(I would like to inform you that I wrote it in English using a translator in advance.)

Hello, SecurityOnion users.

I am currently planning to develop a solution that will compensate for the shortcomings of Security Onion as a personal project.
(It's for studying. I think SecurityOnion is a great OS.)

As a result of my 2 weeks or so of using it, I can't clearly see the shortcomings of a great OS, so I'm asking you this question!

Tell me if there are any shortcomings and complements to Security Onion!

**Comment by Acewiza** [comment]
Compliment:  Security Onion is the greatest network defense system ever deployed.  Hyperbole intended!  

Shortcoming:  Not smart enough to install and operate itself.  Yet.  AI, imagine that...

Sorry to slow you down looking up words.  ;-)
